src/routes/controllers/productsControllers/getProductsByCategory.test.js

```diff
@@ -0,0 +1,164 @@
+import mongoose, { mongo } from 'mongoose';
```

Comment:
what is the purpose of using { mongo }?
you are not using it anywhere in the script.